### ISSUE TITLE
Неправильно

### ISSUE THREAD

**Issue by AnPogrebnyak** [MarharytaTsepiennikova/prog.kiev.ua#2]
Надо написать код, который переставляет элементы массива зеркально, а не только выводит их на экран в обратном порядке.
https://github.com/MarharytaTsepiennikova/prog.kiev.ua/blob/44c2a1dc01d82fdc5607f1c450422738393ef266/Homework3_5_1.java#L5

**Comment by MarharytaTsepiennikova** [comment]
такой вариант: https://github.com/MarharytaTsepiennikova/homework_3/blob/master/src/Homework3_5_1.java

**Comment by AnPogrebnyak** [comment]
Нет. Надо не создать новый массив с перевернутыми значениями, а переставить их в текущем массиве
